syft/pkg/license.go

```diff
@@ -122,13 +129,35 @@ func NewLicenseFromURLs(value string, urls ...string) License {
 	return l
 }
 
+func stripUnwantedCharacters(rawURL string) (string, error) {
+	// Step 1: Remove newline and tab characters
+	cleanedURL := strings.ReplaceAll(rawURL, "\n", "")
+	cleanedURL = strings.ReplaceAll(cleanedURL, "\t", "")
```

Comment:
the trim space call following this section seem sufficient -- unless we're trying to remove embedded newlines/tabs, but that seems unlikely